docs/Configuration.rst

```diff
@@ -5981,7 +6036,166 @@ data is stored in the database.
+
+    .. code-block:: jproperties
+
+          exo.wallet.ContractTransactionVerifierJob.expression=0 0 * ? * * *
+
+   -  Checks all pending transactions sent using wallet application to update its state internally in internal store and to send notifications for receiver and sender.
+      The check is made periodically and can be configured using the following property (Default value: every 10 seconds):
+
+    .. code-block:: jproperties
+
+          exo.wallet.PendingTransactionVerifierJob.expression=0/10 * * * * ?
+
+   -  Send transactions on blockchain using *Admin wallet* that is used to administrate all other wallets.
+      In fact this wallet is used to initialize and send rewards to users and spaces wallets.
+
+- Unlike the *Admin wallet* which uses `Web3j <https://web3j.io/>`__ (Server side) to issue transactions, the HTTP endpoint is used by *users and spaces wallets* using `Web3.js <https://web3js.readthedocs.io/>`__ (Browser side) to communicate with the blockchain to issue transactions.
+
+  |WalletBlockchainCommunication|
+
+Blockchain transaction settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- When sending a transaction to the Ethereum blockchain, some generic properties are required, especially *gas limit* and *gas price*.
+
+  - **Gas limit**
+
+    The *gas limit* is the maximum gas that can take a transaction using the token contract. It can be modified using the following property:
+
+      .. code-block:: jproperties
+
+            exo.wallet.transaction.gas.limit=150000
+
+  - **Gas price**
+
+    The *gas price* parameters will determine transaction fee of operations made using eXo Wallet addon. In the UI, the user has three choices:
+
+      -  Cheap transaction: this option will use the gas price configured by the following property (in `WEI <http://ethdocs.org/en/latest/ether.html>`__):
+
+      .. code-block:: jproperties
+
+            exo.wallet.transaction.gas.cheapPrice=4000000000
+
+      -  Normal transaction: this option will use the gas price configured by the following property (in `WEI <http://ethdocs.org/en/latest/ether.html>`__):
+
+      .. code-block:: jproperties
+
+            exo.wallet.transaction.gas.normalPrice=8000000000
+
+      -  Fast transaction: this option will use the gas price configured by the following property (in `WEI <http://ethdocs.org/en/latest/ether.html>`__):
+
+      .. code-block:: jproperties
+
+            exo.wallet.transaction.gas.fastPrice=15000000000
+
+
+* When a transaction is sent and wasn't mined for several days, it will be marked as failed in eXo Wallet internal database.
+  The maximum number of days waiting for transaction mining can be configured using the following property:
+
+    .. code-block:: jproperties
+
+          exo.wallet.transaction.pending.maxDays=3
+
+
+    .. note:: If the transaction still not mined after ``exo.wallet.transaction.pending.maxDays`` days, it will be marked as failed transaction in internal eXo database.
```

Comment:
```suggestion
    .. note:: If the transaction is still not mined after ``exo.wallet.transaction.pending.maxDays`` days, it will be marked as a failed transaction in eXo's internal database.
```